app/views/docs/command-line.phtml

```diff
@@ -297,7 +297,7 @@ brew install --HEAD appwrite</code></pre>
 <p>To create a new document in an existing collection, use the `createDocument` command.</p>
 
 <div class="ide margin-bottom" data-lang="bash" data-lang-label="CLI">
-    <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite databases createDocument --collectionId <ID> --documentId 'unique()' --data '{ "Name": "Iron Man" }' --read role:all team:abc</code></pre>
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite databases createDocument --collectionId <ID> --documentId 'unique()' --data '{ "Name": "Iron Man" }' --permissions "['read(all)',' 'write(team:abc)'']"</code></pre>
```

Comment:
@christyjacob4 is this the syntax for the CLI array?

Reply:
> @christyjacob4 is this the syntax for the CLI array?

Also interested.

Reply:
No, arrays are supposed to be **space separated**. I believe its already fixed in the pr

`--permissions a b c d`